addons/music/XEH_preInit.sqf

```diff
@@ -0,0 +1,5 @@
+#include "script_component.hpp"
+
+#include "XEH_PREP.sqf"
+
+[] call FUNC(initMusicHandler)
```

Comment:
Don't think a function is needed here. Just put addMusicEH into the preInit script